README.md

```diff
@@ -65,6 +65,7 @@ The first step is to create the workflow in your repository. We describe two way
            uses: calyptia/fluent-linter-action@main
            with:
              CALYPTIA_API_KEY: ${{ secrets.CALYPTIA_API_KEY }}
+             FOLLOW_SYMBOLIC_LINKS: 'true' # this is optional. It defaults to false.
```

Comment:
It defaults to true surely?

Reply:
it defaults to false, it says it on the description. This is an example.

Reply:
I think you commented the opposite below :-) It looks like a change is coming though so will wait.